packages/cdk/lib/roles/cromwell-adapter-role.ts

```diff
@@ -0,0 +1,19 @@
+import * as cdk from "monocdk";
+import * as iam from "monocdk/aws-iam";
+import { BucketOperations } from "../../common/BucketOperations";
+
+export interface CromwellAdapterRoleProps {
+  readOnlyBucketArns: string[];
+  readWriteBucketArns: string[];
+}
+
+export class CromwellAdapterRole extends iam.Role {
+  constructor(scope: cdk.Construct, id: string, props: CromwellAdapterRoleProps) {
+    super(scope, id, {
+      assumedBy: new iam.ServicePrincipal("ecs-tasks.amazonaws.com"),
+    });
+
+    BucketOperations.grantBucketAccess(this, this, props.readOnlyBucketArns, true);
+    BucketOperations.grantBucketAccess(this, this, props.readWriteBucketArns);
```

Comment:
I assume the `true` is a `isReadOnly` parameter? I kind of prefer CDK's `grantReadWrite` and `grantRead` for clarity.

Reply:
Yup its for the readonly parameter, I agree I think it should be more readable. I can push that out in another PR tho since I think it touches a couple other places of code.